packages/react/src/components/tabs/tabs.tsx

```diff
@@ -150,6 +156,27 @@ export const Tabs: VoidFunctionComponent<Props> = ({
     const scrollLeftButtonRef = createRef<HTMLButtonElement>();
     const scrollRightButtonRef = createRef<HTMLButtonElement>();
 
+    const addButton = {
+        ...{
```

Comment:
Nitpick mais ça reviendrait au même de juste avoir les attributs au 1er niveau! :)

Reply:
Je suis pas certain de comprendre? Ici c'est pour m'assurer d'avoir mes defaults.

Est-ce que tu suggère dequoi comme ca?
```suggestion
        const addButton = {
            label: providedAddButtonProps.label || t('addButton'),
            ...
        }
```

Reply:
C'est d'enlever le spread.

```
const addButtonProps = {
        label: t('addTab'),
        disabled: false,
        loading: false,
        tooltipContent: null,
        ...providedAddButtonProps,
    };
```

Reply:
Aaaah ben oui 🤦